OpenDreamRuntime/DreamValue.cs

```diff
@@ -716,5 +770,5 @@ public sealed class DreamValueColorMatrixSerializer : ITypeReader<ColorMatrix, D
             return new(source);
         }
     }
-    #endregion Serialization
```

Comment:
This looks accidental, there's still `#region Serialization` above and I don't see why you'd remove this.